components/ActionMenu/__snapshots__/ActionMenu.stories.storyshot

```diff
@@ -33,7 +33,7 @@ exports[`Storyshots ActionMenu Default 1`] = `
         className="MuiButton-label"
       >
         <svg
-          aria-hidden="true"
+          aria-hidden={true}
           className="MuiSvgIcon-root"
           focusable="false"
```

Comment:
Does storybook forget to change `focusable="false"` to `focusable={false}` ?

Reply:
Personally I don't know why this would change XD
For DOM elements, `{true}` and `"true"` are probably identical